monitoring/src/utils/config-helper.ts

```diff
@@ -0,0 +1,56 @@
+import type { AwsRegionOpt, JurisdictionOpt } from '../config';
+
+export const AWS_REGIONS = {
+	EU_WEST_1: 'eu-west-1',
+	US_WEST_1: 'us-west-1',
+	CA_CENTRAL_1: 'ca-central-1',
+	AP_SOUTHEAST_2: 'ap-southeast-2',
+};
+
+export const JURISDICTIONS = {
+	TCFV2: 'tcfv2',
+	CCPA: 'ccpa',
+	AUS: 'aus',
+};
+
+export class ConfigHelper {
```

Comment:
Does this need to be a class or can we export the functions without the class wrapper?